toolchains/syno-armada38x-5.2/Makefile

```diff
@@ -0,0 +1,16 @@
+TC_NAME = syno-$(TC_ARCH)
+
+TC_ARCH = armada38x
+TC_VERS = 5.2
+TC_FIRMWARE = 5.2-5644
+
+TC_DIST_NAME = armada38x-gcc493_glibc220_hard-GPL.txz
+
+TC_EXT = txz
+TC_DIST_SITE = http://sourceforge.net/projects/dsgpl/files/DSM%205.2%20Tool%20Chains/Marvell%20Armada%2038x%20Linux%203.10.35
+
+TC_BASE_DIR = arm-marvell-linux-gnueabi
```

Comment:
Value should be `arm-unknown-linux-gnueabi` for the next three lines. This follows the dirnames you see after extracting.